.github/CODEOWNERS

```
@@ -0,0 +1 @@
+/caret/ @takam5f2
```

As a global owner. (If someone else wants, they can also be a global owner.)

```suggestion
* @kenji-miyake
/caret/ @takam5f2
```

---

Or,

```suggestion
* kenji.miyake@tier4.jp
/caret/ takayuki.akamine@tier4.jp
```